lonboard/geopandas.py

```diff
@@ -0,0 +1,296 @@
+            alpha : Alpha (opacity) parameter in the range (0,1) passed to
+                mapclassify.util.get_color_array.
+            layer_kwargs : Additional keyword arguments passed to lonboard.viz layer
+                arguments (either polygon_kwargs, scatterplot_kwargs, or path_kwargs,
+                depending on input geometry type).
+            map_kwargs : Additional keyword arguments passed to lonboard.viz map_kwargs.
+            classification_kwds : Additional keyword arguments passed to
+                `mapclassify.classify`.
+            nan_color : Color used to shade NaN observations formatted as an RGBA list.
+                Defaults to [255, 255, 255, 255]. If no alpha channel is passed it is
+                assumed to be 255.
+            color : single or array of colors passed to Layer.get_fill_color
+                or a lonboard.basemap object, or a string to a maplibre style basemap.
+            vmin : Minimum value for color mapping.
+            vmax : Maximum value for color mapping.
+            wireframe : Whether to use wireframe styling in deckgl.
+            tiles : Either a known string {"CartoDB Positron",
+                "CartoDB Positron No Label", "CartoDB Darkmatter",
+                "CartoDB Darkmatter No Label", "CartoDB Voyager",
+                "CartoDB Voyager No Label"}
+            highlight : Whether to highlight each feature on mouseover (passed to
+                lonboard.Layer's auto_highlight). Defaults to False.
+            m: An existing Map object to plot onto.
+
+        Returns:
+        lonboard.Map
+            a lonboard map with geodataframe included as a Layer object.
+        """
+        return _dexplore(
+            self._obj,
+            column,
+            cmap,
+            scheme,
+            k,
+            categorical,
+            elevation,
+            extruded,
+            elevation_scale,
+            alpha,
+            layer_kwargs,
+            map_kwargs,
+            classification_kwds,
+            nan_color,
+            color,
+            vmin,
+            vmax,
+            wireframe,
+            tiles,
+            highlight,
+            m,
+        )
+
+
+def _dexplore(
+    gdf,
+    column=None,
+    cmap=None,
+    scheme=None,
+    k=6,
+    categorical=False,
+    elevation=None,
+    extruded=False,
+    elevation_scale=1,
+    alpha=1,
+    layer_kwargs=None,
+    map_kwargs=None,
+    classification_kwds=None,
+    nan_color=[255, 255, 255, 255],
+    color=None,
+    vmin=None,
+    vmax=None,
+    wireframe=False,
+    tiles="CartoDB Darkmatter",
+    highlight=False,
+    m=None,
+):
+    """explore a dataframe using lonboard and deckgl.
+
+    See the public docstring for detailed parameter information
+
+    Returns
+    -------
+    lonboard.Map
+        a lonboard map with geodataframe included as a Layer object.
+
+    """
+
+    providers = {
```

Comment:
A minor note on inconsistency that may bite some users - geopandas is able to consume the strings representing tile names in any formatting xyzservices is capable of parsing. So both `cartodbpositron` and `CartoDB Positron` result in the same tiles.

Reply:
> xyzservices is capable of parsing

Do you know how that parsing works? Does it remove any whitespace and then convert to lower case?

Reply:
interesting, i had no idea those things work 😆